packages/driver/src/cy/actionability.ts

```diff
@@ -53,9 +54,127 @@ const getPositionFromArguments = function (positionOrX, y, options) {
   return { options, position, x, y }
 }
 
+const ensureElDoesNotHaveCSS = ($el, cssProperty, cssValue, name: string, onFail) => {
```

Comment:
Several methods from ensures.ts are only used here, and don't really make sense to expose for people to use in queries. I've moved them into this file.